src/main/kotlin/no/nav/medlemskap/inst/lytter/service/JoarkService.kt

```diff
@@ -21,9 +23,10 @@ class JoarkService(
     }
     suspend fun handle(record : MedlemskapVurdertRecord)
     {
-        if (validateRecord(record)){
+        val medlemskapVurdering = JaksonParser().parseToObject(record.json)
+        if (validateRecord(medlemskapVurdering) && medlemskapVurdering.datagrunnlag.ytelse == Ytelse.LOVME_GCP.name){
```

Comment:
Savner også en test som vi bruke for å validere .. lage en egen funksjon som heter noe alla (må ikke hete det) skalOpprettePdf(medlemskapVurdering)..
Denne bør være relariv enkel å teste uten å måtte ta i bruk mock osv..